components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           imagePullPolicy: Always
           command:
             - /manager
-          args: ["--oauth-proxy-image", "registry.redhat.io/openshift4/ose-oauth-proxy:v4.10"]
+          args: ["--oauth-proxy-image", "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46"]
```

Comment:
we need to, together with upstream folks, agree on a version of ose-oauth-proxy and the associated manifest list digest.
Openshift 4.10 is not recent anymore.
Question is: what is currently recommended upstream and downstream? Openshift 4.14 or 4.15?
Can you ask that in the general RHOSAI and Opendatahub teams, please?
https://access.redhat.com/support/policy/updates/openshift#dates
Now, I did not have any problems in the past running ose-oauth-proxy in 4.10 or 4.11 or 4.12 version on e.g. Openshift 4.14 or 4.15. However, recently, I updated to ose-oauth-proxy:v4.14 cause that is our current environment ... to avoid any problems and used the associated manifest list digest.

https://catalog.redhat.com/software/containers/openshift4/ose-oauth-proxy/5cdb2133bed8bd5717d5ae64?architecture=amd64&image=6643eda9c58e057a0a68e3aa

The latest assembly stream build is from July 2 for tag v4.14 and the associated manifest list digest is

sha256:2f9603d72bad86ae8cc51d0f9677acfcde77ada268dc28675420793223ba56e2

Reply:
absolutely, creating https://issues.redhat.com/browse/RHOAIENG-10827 for that, and here I'll just use the obsolete version what red-hat-data-services fork of kubeflow uses